.github/workflows/ClimaLandSimulations.yml

```diff
@@ -0,0 +1,34 @@
+name: ClimaLandSimulations CI
+on:
+  pull_request:
+
+jobs:
+  lib-climalandsimulations:
+    runs-on: ubuntu-20.04
+    timeout-minutes: 45
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - uses: julia-actions/setup-julia@latest
+        with:
+          version: '1.10'
+      - uses: actions/cache@v3
+        env:
+          cache-name: cache-artifacts
+        with:
+          path: ~/.julia/artifacts
+          key: ${{ runner.os }}-test-${{ env.cache-name }}-${{ hashFiles('**/Project.toml') }}
+          restore-keys: |
+            ${{ runner.os }}-test-${{ env.cache-name }}-
+            ${{ runner.os }}-test-
+            ${{ runner.os }}-
+      - name: Install Julia dependencies
+        run: >
+          julia --project=monorepo -e 'using Pkg; Pkg.develop(path="$(pwd())"); Pkg.develop(path="$(pwd())/lib/ClimaLandSimulations")'

```

Comment:
Why the `monorepo` project?

Reply:
I am not sure... It is what ClimaCore.jl does for its ci of projects in /lib folders, but I am not sure what it is for or if it is needed.  https://github.com/CliMA/ClimaCore.jl/blob/main/.github/workflows/ClimaCoreMakie.yml#L28

Reply:
I don't know why it's there, but you don't need it.